server/views/formBuilder.njk

```diff
@@ -209,7 +209,7 @@
                 {% for item in formValues %}
                     {# When submitting an empty form, the formValues is an array containing an empty string #}
                     {# This is to cater for other input types which are part of the Add-Another pattern #}
-                    {% if item.length > 1 %}
+                    {% if (item.length > 1) and (additionalCondition.uploadSummary[0].thumbnailImage) %}
```

Comment:
not sure why this is `0` rather than `i`?

Reply:
This is existing code present before the change. It looks like FarSight intended the uploadSummary to contain multiple files against a condition. But then hardcoded reference to multiple exclusions zones to look for the first index, as its only one file per condition.

The problem with uploading more than one file against the same condition relates to the other data belonging to the condition. The file name region is a text area. So you cannot uploads more than one file using this method, because the file name region can only be added once.